Example/GiniVision/SelectAPIViewController.swift

```diff
@@ -92,6 +92,13 @@ class SelectAPIViewController: UIViewController {
         giniConfiguration.debugModeOn = true
         giniConfiguration.fileImportSupportedTypes = .pdf_and_images
         giniConfiguration.navigationBarItemTintColor = UIColor.white
+        giniConfiguration.customDocumentValidations = { document in
```

Comment:
Great idea! What if they want to do an async validation? For ex. doing some complex validation that would block the main thread?